pkgs/development/python-modules/safe-pysha3/default.nix

```
@@ -0,0 +1,19 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder }:
```

```suggestion
{ lib
, buildPythonPackage
, fetchPypi
, pythonOlder
, unittestCheckHook
}:
```

One per line and you're going to need `unittestCheckHoo` to run the tests.

---

The tests are run by `setuptoolsCheckHook` implicitly.

---

Which is deprecated, the log should tell you as much. We only support explicit testing.

---

Added back.